src/xqerl_code_server.erl

```diff
@@ -643,18 +643,14 @@ init_rest(DispatchFileName) ->
                 ok = write_dispatch(DispatchFileName, Dis),
                 Dis
         end,
-    DisTerm = xqerl_restxq:endpoint_sort(Paths),
-    Dispatch = cowboy_router:compile([{'_', DisTerm}]),
+    Routes = lists:flatten(
+        [ xqerl_restxq:endpoint_sort(Paths),
+          {"/assets/[...]", cowboy_static, {priv_dir, xqerl, "static/assets"}} ]),       
+    Dispatch = cowboy_router:compile([{'_', Routes}]),
     _ = cowboy:start_clear(
         xqerl_listener,
         [{port, Port}],
-        #{
-            env => #{dispatch => Dispatch},
-            stream_handlers => [
-                cowboy_compress_h,
-                cowboy_stream_h
-            ]
```

Comment:
Are the stream handlers dropped on purpose? I vaguely remember this causing problems with gzip and large payloads.

Then again, I may be remembering incorrectly :-)

Reply:
Yep, according to 2.9 docs, they are the default

Reply:
> I vaguely remember this causing problems with gzip and large payloads

You are most likely remembering this from
https://ninenines.eu/docs/en/cowboy/2.8/manual/cowboy_compress_h/
re.  `cowboy_compress_h` 
  > Because the file must be read in memory to be compressed, this module is not suitable for automatically compressing large files.

Reply:
Thanks!